airbyte-integrations/connectors/source-xero/manifest.yaml

```diff
@@ -1612,35 +1612,26 @@ definitions:
     type: HttpRequester
     url_base: https://api.xero.com/api.xro/2.0/
     authenticator:
-      type: SelectiveAuthenticator
-      authenticators:
-        oauth2_access_token:
-          type: BearerAuthenticator
-          api_token: "{{ config['credentials']['access_token'] }}"
-        oauth2_confidential_application:
-          type: OAuthAuthenticator
-          scopes:
-            - offline_access
-            - accounting.transactions.read
-            - accounting.reports.read
-            - accounting.budgets.read
-            - accounting.reports.tenninetynine.read
-            - accounting.journals.read
-            - accounting.settings.read
-            - accounting.contacts.read
-            - accounting.attachments.read
-            - assets.read files.read projects.read
-            - openid
-          client_id: "{{ config['credentials']['client_id'] }}"
-          grant_type: client_credentials
-          client_secret: "{{ config['credentials']['client_secret'] }}"
-          expires_in_name: expires_in
-          access_token_name: access_token
-          refresh_request_body: {}
-          token_refresh_endpoint: https://identity.xero.com/connect/token
-      authenticator_selection_path:
-        - credentials
-        - auth_type
+      type: OAuthAuthenticator
+      client_id: "{{ config['client_id'] }}"
+      client_secret: "{{ config['client_secret'] }}"
+      access_token_value: "{{ config['client_access_token'] }}"
```

Comment:
The `access_token_value` should be provided ONLY when the only key that has been edited out of the `OAuthFlow` is `access_token` (meaning that there is no `refresh_token` available from the API perspective). 

Please remove this property from `OAuthAuthenticator`.